src/style/button.css

```diff
@@ -0,0 +1,24 @@
+.product__button {
+  display: flex;
+  justify-content: space-evenly;
+  border: 1px solid var(--color--blue);
+  height: 40px;
+  width: 166px;
+  border-radius: 5px;
+  background-color: var(--color--blue);
+  cursor: pointer;
+  
+  align-items: center;
+  font-family: "Roboto", sans-serif;
```

Comment:
Please don't repeat this rule. Remove from this class and Add to body selector. It's for all cases

Reply:
i did